pkgs/desktops/gnome/extensions/no-title-bar-forked/default.nix

```diff
@@ -1,7 +1,7 @@
 { lib, stdenv, fetchFromGitHub, substituteAll, glib, gettext, xorg }:
 
 stdenv.mkDerivation rec {
-  pname = "gnome-shell-extension-no-title-bar";
+  pname = "gnome-shell-extension-no-title-bar-forked";
```

Comment:
This also sounds like something we should not need.

Reply:
So my point is, the manually packaged extensions must somehow match the automatically packaged ones in order for overriding to work.

Reply:
Could we do it in the other direction – rename the automatically packaged extension instead? Since the names are arbitrary, we might as well use ones that make sense.

Reply:
Middle ground: I'd like to keep the `pname` the same, but I'd be fine with renaming the attribute name of the extensions (pname is almost worthless anyways).

The most promising approach at implementing this would be to key all manually packaged extensions with their UUID as well, so that I can first do the merge/override, and then give them proper names.

Reply:
`pname` is what Repology uses so we should keep it somewhat canonical and reasonable.

I would say the easiest would be adding a section for changing bad slugs in `extensionRenames.nix`. But adding detection of duplicates based on UUID would be useful when people add a new manually packaged extension so that they choose a correct name.